packages/language/test/fourslash/validate/IBM2615I/do-iterative-forever.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+/// <reference path="../../framework.ts" />
+
+/**
+ * DO FOREVER – doType4 – no warning
+ */
+// @wrap: main
+//// <|1:DO FOREVER|>;
```

Comment:
Minor: Even if the error were to appear, we would only show it on the `DO` keyword.
```suggestion
//// <|1:DO|> FOREVER;
```